docker/lwua-ingest/lwua-py/lwua/watcher.py

```diff
@@ -0,0 +1,84 @@
+import time
+from datetime import datetime
+import os
+from pathlib import Path
+from dotenv import load_dotenv
+from abc import ABC, abstractmethod
+import logging
+from lwua.helpers import enable_logging, resolve_path
+from lwua.graphdb import (
```

Comment:
dependency on graph is coming from the observer implementation in this file

however better to get that out into either separate file, or back to ingest.py